css/index.css

```
@@ -52,7 +52,7 @@ select {
   position: relative;
   top: 100px;
   width: 100%;
-  height: 1200px;
+  height: 600px;
```

height moet op 1200px
```suggestion
  height: 1200px;
```